openy_af4_vue_app/src/main.js

```diff
@@ -35,7 +46,7 @@ document.addEventListener('openy_activity_finder_event', e => {
   }
 })
 
-Vue.config.productionTip = false
+Vue.config.productionTip = false;
```

Comment:
is is dev leak?

Reply:
@podarok sorry, I'm not sure that I got what you meant here
It's just an option to prevent the production tip on Vue startup (https://v2.vuejs.org/v2/api/#productionTip_